File: Python/List-2/sum13.py
"""
Dada uma lista numérica "nums", retorne a soma de seus elementos (0 caso seja vazia).
Porém, se houver um "13" ele deverá ser ignorado, assim como o número
imediatamente a sua frente.
"""


def sum13(nums):
    soma = 0
    for i in range(len(nums)):
        if nums[i:i+1] != [13] and nums[i-1:i] != [13]:
            soma += nums[i]
    return soma


print(sum13([1, 2, 13, 2, 1, 13]))

File: Java/Array-2/sum13/Sum13.java
package sum13;

public class Sum13 {
    //Retorne a soma dos números contidos no Array ("nums"), caso não tenha retorne 0. Contudo, se houver um "13", ele
    //não deverá ser considerado e nem o que lhe sucede.

    public int sum13(int[] nums) {
        int count = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] == 13){
                i++;
                continue;
            }
            count += nums[i];
        }
        return count;
    }
}
